smart_open/s3.py

```diff
@@ -527,6 +565,10 @@ def to_boto3(self):
     #
     # Internal methods.
     #
+    @property
```

Comment:
Do we actually need this property?

It's internal, so not subject to any contracts. Our internal code can use `self._raw_reader._content_length` directly, right?